app/helpers/holdings_helper.rb

```diff
@@ -44,7 +44,7 @@ def links(name)
 
     # :links will get all the links, at this point we want just the one that matches
     # the name we passed or the free resource
-    holdings(@document, :links).inject(:merge).select do |k|
+    @urls&.inject(:merge)&.select do |k|
```

Comment:
I think the :links symbol is no longer defined. It would be a minor documentation change.

Reply:
Thanks @lagoan.  I've updated the comment to reflect the change.